### DISCUSSION TITLE
阿里云盘webdav这两天无法访问

### DISCUSSION THREAD

**Discussion by oyal** [AlistGo/alist#6826] [General]
potplayer 和 nplayer 都无法播放，然后升级了最新 alist 3.35.0 也是一样。网页端是正常可以播放的

> 补充：昨天是有的可以播放有的不能播放，刚才看了一下好像是都不能播放了

![2024-07-20_13-56-03](https://github.com/user-attachments/assets/c4557f55-5738-47ad-9b72-72e9f80f32d0)

**Comment by ririye** [comment]
正常，垃圾阿里开始搞第三方了，开始对第三方收费了
消息称阿里云盘第三方播放器高清点播和 WebDAV 将需要额外付费https://baijiahao.baidu.com/s?id=1797203873637595586&wfr=spider&for=pc

**Reply by oyal** [reply]
已经上线了

![Screenshot_2024-07-20-15-38-31-765_com.alicloud.databox.jpg](https://github.com/user-attachments/assets/a243ca31-1f1f-49c3-9bb7-c3e7b3fbd354)

**Comment by verydemo** [comment]
就算买了第三方，也是有高速下载流量限制的， 阿里云盘算是废了；   以后用什么替代呢？还有两年svip 算是打水漂了

**Comment by AspirantH** [comment]
当时宣传的不限速，现在连SVIP都限速，是不是可以告阿里虚假宣传啊？

**Reply by ririye** [reply]
告了也没用，现在阿里后台姓习，阿里就是习党养的抽百姓骨髓的狗而已
